tasks/xpack/security/elasticsearch-xpack-activation.yml

```diff
@@ -15,6 +16,25 @@
     license_activated.status != 200 or
     license_activated.json.license_status is not defined or
     license_activated.json.license_status != 'valid'
+  when: es_version is version('6.9', operator='<=')
+
+- name: Activate ES license (with security authentication)
```

Comment:
This duplicate task can be removed by doing the same trick you did above to determine whether to use `_xpack/license` or just `_license`.